cp-ksqldb-server/include/etc/confluent/docker/configure

```diff
@@ -28,5 +28,18 @@ then
   fi
 fi
 
-dub template "/etc/confluent/docker/ksql-server.properties.template" "/etc/${COMPONENT}/${COMPONENT}.properties"
+# KSQL_CONNECT_GROUP_ID is a required configuration for running embedded connect,
+# so we can proxy it to check whether or not to start an embedded connect worker
+if ! [[ -z "${KSQL_CONNECT_GROUP_ID:=}" ]]; then
+  echo "===> Enabling Embedded Connect"
+  export KSQL_KSQL_CONNECT_WORKER_CONFIG="/etc/${COMPONENT}/connect.properties"
+
+  # Default to 8083, which matches the mesos-overrides. This is here in case we extend the containers to remove the mesos overrides.
+  if [ -z "${KSQL_CONNECT_REST_PORT:=}" ]; then
+    export KSQL_CONNECT_REST_PORT=8083
```

Comment:
I think we need to change the `run` script to `source /etc/confluent/docker/configure`. I still don't think these exports in this file would be picked up when launching the server since `configure` is exporting these variables in a sub-shell of the `run` script. If we source the configure file in the `run` script, the shell that's executing `run` should pick up the env variables properly

Reply:
The change I made for the log4j env variable was moving the export from `configure` to `launch`

Reply:
the change doesn't need to be picked up in the launch (unlike the change for LOG4J). The environemnt variable only needs to be present for the `dub` command below in the `configure` script

Reply:
(I tested this to make sure that it works with docker-compose file without the env variable set)

Reply:
Oh I see, I thought it needed to be present during launch